kusto/kusto.go

```diff
@@ -119,7 +123,15 @@ type Client struct {
 type Option func(c *Client)
 
 // New returns a new Client. endpoint is the Kusto endpoint to use, example: https://somename.westus.kusto.windows.net .
-func New(endpoint string, auth Authorization, options ...Option) (*Client, error) {
```

Comment:
Doesn't this change effectively remove the ability to use ADAL Authorization?
If so, why keep it in the code?

Reply:
Proceeding to remove the corresponding code...